### DISCUSSION TITLE
Launching commands in split tabs

### DISCUSSION THREAD

**Discussion by laeijebor** [Eugeny/tabby#8033] [Q&A]
Hi folks,

My development workflow commonly starts with opening three split tabs, let's say:

- One split for the server.     (`yarn run dev` in `project/server/`)
- One for a web server / bundler.   (`yarn run dev` in `project/web-server/`)
- One split for running Expo / a simulator.  (`expo start` in `project/expo/`)

Has anyone found an elegant way of launching multiple split tabs from one command or hotkey?

I've achieved something similar with teamocil/itermocil on iTerm, or custom commands in Kitty, but am wanting to move fully over to Tabby for work and hobby projects, so this would be a great timesaver!

Also, I'm aware of Tabby Workspace Manager, but I don't really want to cycle between the separate tabs (the 3 splits is a simplification, there are often lots of different services I run for a particular project). https://github.com/composer404/tabby-workspace-manager

**Comment by Eugeny** [comment]
Once you have your splits arranged the way you like it, right-click the tab and select "Save layout as a profile"

**Reply by laeijebor** [reply]
Thanks Eugeny - unless I'm doing something wrong, that just saves the layout, not the command that should be run in each split? If that's not the expected behaviour, I can do some investigation and raise a bug.

Tabby is awesome!

**Reply by Eugeny** [reply]
Correct - Tabby just saves each split's profile. For each of your command, you'll need to create a separate profile that launches that command instead of just a shell
